src/sass/components/_special.scss

```
@@ -0,0 +1,94 @@
+.special {
+  position: relative;
+  &-bg {
+    img {
+      height: 750px;
+      width: 100%;
+    }
+  }
+  &-container {
+    max-width: 100%;
+  }
+  &-product {
```

```suggestion
  position: relative;
  
  &-bg {
    img {
      height: 750px;
      width: 100%;
    }
  }
  
  &-container {
    max-width: 100%;
  }
  
  &-product {
```
tách nhau ra nhìn cho dễ em nhé